src/utils/triggers.js

```diff
@@ -0,0 +1,11 @@
+const getKonnectorFromTrigger = trigger => {
+  if (trigger.message && trigger.message.konnector) {
+    return trigger.message.konnector
+  } else if (trigger.message.Data) {
+    // Legacy triggers
+    const message = JSON.parse(atob(trigger.message.Data))
+    return message.konnector
+  }
+}
+
+export { getKonnectorFromTrigger }
```

Comment:
Do you think I should put this in cozy-doctypes/triggers ?

Reply:
I don't think this is related to doctypes, so I would keep it on banks, at least for now.